### PR TITLE
remove xfce-4.10~precise ppa

### PR DESCRIPTION
**Pull request by DennisLfromGA**
Remove old and now non-existant ppa for xfce4.10~precise.

### CODE REVIEW

File: targets/xfce [hunk 1]
```diff
@@ -9,19 +9,15 @@ CHROOTBIN='crouton-noroot startxfce4'
...
-# Bring in Xfce 4.10 on Ubuntu 12.04
-deb http://ppa.launchpad.net/xubuntu-dev/xfce-4.10/ubuntu $RELEASE main
-deb-src http://ppa.launchpad.net/xubuntu-dev/xfce-4.10/ubuntu $RELEASE main
-EOF
-    # Update database
-    apt-get update || true
+    if [ -s /etc/apt/sources.list.d/xfce-4.10.list ]; then
+        rm /etc/apt/sources.list.d/xfce-4.10.list*
+        # Remove PPA key
+        apt-key del 142986CE
```
Comment (dnschneid) [inline targets/xfce]: I'd rather not remove the key in the event that the user added another xubuntu-dev ppa (such as the 4.12 one).

File: targets/xfce [hunk 2]
```diff
@@ -9,19 +9,15 @@ CHROOTBIN='crouton-noroot startxfce4'
...
-    # Add the software source
-    cat > '/etc/apt/sources.list.d/xfce-4.10.list' <<EOF
-# Bring in Xfce 4.10 on Ubuntu 12.04
-deb http://ppa.launchpad.net/xubuntu-dev/xfce-4.10/ubuntu $RELEASE main
-deb-src http://ppa.launchpad.net/xubuntu-dev/xfce-4.10/ubuntu $RELEASE main
-EOF
-    # Update database
-    apt-get update || true
+    if [ -s /etc/apt/sources.list.d/xfce-4.10.list ]; then
+        rm /etc/apt/sources.list.d/xfce-4.10.list*
```
Comment (dnschneid) [inline targets/xfce]: -f

File: targets/xfce (hunk 1 above)
Comment (DennisLfromGA) [inline targets/xfce]: I wasn't sure about the key removal, makes a lot of sense to leave it.

File: targets/xfce (hunk 2 above)
Comment (DennisLfromGA) [inline targets/xfce]: Of course...

**Pull request merged**